community-operators/jaeger/1.22.1/jaeger.v1.22.1.clusterserviceversion.yaml

```
@@ -0,0 +1,508 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "jaegertracing.io/v1",
+          "kind": "Jaeger",
+          "metadata": {
+            "name": "jaeger-all-in-one-inmemory"
+          }
+        }
+      ]
+    capabilities: Deep Insights
+    categories: Logging & Tracing
+    certified: "false"
+    containerImage: docker.io/jaegertracing/jaeger-operator:1.21.2
```

```suggestion
    containerImage: docker.io/jaegertracing/jaeger-operator:1.22.1
```
@rubenvp8510 Please fix container version so ti is same as it is used in the CSV